jinja2cli/cli.py

```diff
@@ -197,6 +197,12 @@ def cli(opts, args):
             ext = 'jinja2.ext.' + ext
         extensions.append(ext)
 
+    # Use only a specific section if needed
+    if opts.section:
+        section = opts.section
+        if section in data:
```

Comment:
Yeah, that's fine as well. My point is that this should be surfaced to the user instead of just quietly discarded.